README.rst

```diff
@@ -263,9 +226,7 @@ However, Rekall profiles have to be created for each kernel version, and therefo
 update is made to the kernel, the profile has to be re-generated, thus it's a bit less stable
 as the standard LibVMI configuration entries.
 
-Rekall is available at https://github.com/google/rekall. You will also need to install libjson-c-dev
-from your distribution's repository or compile it from source that can be found at
-https://github.com/json-c/json-c.
+Rekall is available at https://github.com/google/rekall.
 
 To create a Rekall profile for Windows you need to determine the PDB filename and GUID of the
```

Comment:
You can rewrite the rest of this section and simply redirect to `libvmi/tools/windows-offset-finder` `README`